package.json

```diff
@@ -10,8 +10,7 @@
     "changelog": "./bin/run-changelog.sh",
     "coverage": "./bin/run-test.sh -c",
     "bump": "./bin/run-bump.sh",
-    "closure": "./node_modules/.bin/closure-npc",
-    "commitmsg": "commitlint -e"
+    "closure": "./node_modules/.bin/closure-npc"
```

Comment:
Just since it's highlighted in this change, should we remove this at some point?

Reply:
Done.